packages/framework/src/Resources/views/Debug/Elasticsearch/template.html.twig

```diff
@@ -0,0 +1,139 @@
+{% extends '@WebProfiler/Profiler/layout.html.twig' %}
+
+{% block toolbar %}
+    {% if collector.data.requestsCount > 0 %}
+        {% if collector.data.totalRequestsTime * 1000 > 200 %}
+            {% set status = 'red' %}
+        {% elseif collector.data.totalRequestsTime * 1000 > 100 %}
```

Comment:
`collector.data.totalRequestsTime` is always multiplied by 1000 in the template. I suggest to move this multiplication to the source of these data (ElasticsearchCollector, eventually ElasticsearchDebugStack if you accept moving methods idea)

Reply:
or maybe it could be done like `        {% if collector.data.totalRequestsTime > 0.2 %}`